clang-tools-extra/test/clang-doc/test-path-abs.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: echo "CHECK: var RootPath = \"%/t/docs\";" > %t/check.txt
+// RUN: cp "%s" "%t/test.cpp"
+// RUN: clang-doc --format=html --executor=standalone -p %t %t/test.cpp --output=%t/docs
```

Comment:
```suggestion
// RUN: clang-doc --format=html --executor=standalone  %s --output=%t
```
I don't think you need `-p`, and I don't see a good reason why you need a subdirectory for `docs`.